File: .claude/skills/cli-builder-workspace/iteration-2/eval-1-python-project-setup/without_skill/outputs/src/projinit/banner.py
"""ASCII banner display for projinit."""

from rich.console import Console
from rich.text import Text
from rich.panel import Panel
from rich.align import Align

BANNER_ART = r"""
[bold cyan]  ██████╗ ██████╗  ██████╗      ██╗██╗███╗   ██╗██╗████████╗[/]
[bold cyan]  ██╔══██╗██╔══██╗██╔═══██╗     ██║██║████╗  ██║██║╚══██╔══╝[/]
[bold blue]  ██████╔╝██████╔╝██║   ██║     ██║██║██╔██╗ ██║██║   ██║[/]
[bold blue]  ██╔═══╝ ██╔══██╗██║   ██║██   ██║██║██║╚██╗██║██║   ██║[/]
[bold magenta]  ██║     ██║  ██║╚██████╔╝╚█████╔╝██║██║ ╚████║██║   ██║[/]
[bold magenta]  ╚═╝     ╚═╝  ╚═╝ ╚═════╝  ╚════╝ ╚═╝╚═╝  ╚═══╝╚═╝   ╚═╝[/]
"""

TAGLINE = "[dim]━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━[/dim]"
SUBTITLE = "[bold white]  Project Scaffolding CLI[/bold white]  [dim]v1.0.0[/dim]"
DECORATIVE = "[dim]  Kickstart your next project in seconds.[/dim]"


def display_banner(console: Console | None = None) -> None:
    """Display the startup banner with styling."""
    if console is None:
        console = Console()

    console.print()
    console.print(BANNER_ART)
    console.print(TAGLINE)
    console.print(SUBTITLE)
    console.print(DECORATIVE)
    console.print(TAGLINE)
    console.print()

File: .claude/skills/cli-builder-workspace/iteration-1/eval-0-nodejs-csv-converter/without_skill/outputs/src/ui/banner.ts
import chalk from 'chalk';

export const displayBanner = (): void => {
  const banner = `
${chalk.cyan.bold('  ██████ ███████ ██    ██ ██████      ██ ███████  ██████  ███    ██')}
${chalk.cyan.bold(' ██      ██      ██    ██      ██     ██ ██      ██    ██ ████   ██')}
${chalk.cyan.bold(' ██      ███████ ██    ██  █████      ██ ███████ ██    ██ ██ ██  ██')}
${chalk.cyan.bold(' ██           ██  ██  ██  ██     ██   ██      ██ ██    ██ ██  ██ ██')}
${chalk.cyan.bold('  ██████ ███████   ████   ███████  █████  ███████  ██████  ██   ████')}

${chalk.gray('  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━')}
${chalk.white.bold('  CSV to JSON Converter')} ${chalk.gray('v1.0.0')}
${chalk.gray('  ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━')}
`;

  console.log(banner);
};
